src/libsync/configfile.cpp

```diff
@@ -103,6 +103,11 @@ static constexpr char moveToTrashC[] = "moveToTrash";
 
 static constexpr char certPath[] = "http_certificatePath";
 static constexpr char certPasswd[] = "http_certificatePasswd";
+
+bool validUpdateChannel(const QString &channel)
+{
+    return channel == QStringLiteral("stable") || channel == QStringLiteral("beta");
```

Comment:
why not using a const list that would be easier to extend in the future ?

Reply:
Replaced this method with a const QSet in the anonymous namespace